pk_tic_tac_toe.rb

```diff
@@ -31,50 +42,39 @@ def initialize(name, game_piece)
 end
 
 board = TicTacToeGame.new
-player1 = Player
-player2 = Player
 
 def display_board(board)
+  puts ''
   puts " #{board.board[0]} | #{board.board[1]} | #{board.board[2]} "
   puts '---x---o---'
   puts " #{board.board[3]} | #{board.board[4]} | #{board.board[5]} "
   puts '---o---x---'
   puts " #{board.board[6]} | #{board.board[7]} | #{board.board[8]} "
-end
-
-def create_player1(player1)
-  print 'Hello player 1, please enter your first name: '
-  name = gets.chomp.capitalize
-  puts "Welcome, #{name}; your game piece will be 'X'."
   puts ''
-  game_piece = 'X'
-  player1.new(name, game_piece)
 end
 
-def create_player2(player2)
-  print 'Hello player 2, please enter your first name: '
-  name = gets.chomp.capitalize
-  puts "Welcome, #{name}; your game piece will be 'O'."
+def create_player(player_num, game_piece)
+  print "Hello player #{player_num} please enter your first name: "
+  name = gets.strip.capitalize
+  puts "Welcome, #{name}; your game piece will be #{game_piece}"
   puts ''
-  game_piece = 'O'
-  player2.new(name, game_piece)
+  Player.new(name, game_piece)
```

Comment:
I now instantiate a new player object by directly calling the `new` class method on the `Player` class.